tests/tacacs/test_ro_user.py

```diff
@@ -77,6 +78,22 @@ def wait_for_tacacs(localhost, remote_ip, username, password):
                 current_attempt += 1
 
 
+def ssh_remote_run_with_ipv6(localhost, dutip, ptfhost, user, password, command):
+    retry_count = RETRY_COUNT
```

Comment:
>retry_count [](http://example.com/codeflow?start=4&length=11)

`retry_count` could be a function parameter.

Reply:
Fixed